src/style.css

```diff
@@ -1,3 +1,60 @@
 body {
   margin: 0;
+  --visited-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+}
+
+.nav_list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav_item {
+  display: block;
+  margin-right: 20px;
+  text-decoration: none;
+  color: #000;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  height: 60px;
+  line-height: 60px;
```

Comment:
not a good approach to center a text with `line-height`. better to use flexbox for such cases